src/ripple/protocol/jss.h

```diff
@@ -361,6 +363,12 @@ JSS(invalid_API_version);  // out: Many, when a request has an invalid
 JSS(io_latency_ms);        // out: NetworkOPs
 JSS(ip);                   // in: Connect, out: OverlayImpl
 JSS(is_burned);            // out: nft_info (clio)
+JSS(isSerialized);         // out: RPC server_definitions
+                           // matches server_definitions.json format
+JSS(isSigningField);       // out: RPC server_definitions
+                           // matches server_definitions.json format
+JSS(isVLEncoded);          // out: RPC server_definitions
+                           // matches server_definitions.json format
```

Comment:
this comment is not very informative. it would help if you could describe this field, where and why its being used

Reply:
What more info would you like? I think all that info is already there, to the same degree that explanations exist elsewhere in the file.

Reply:
this is not a binding requirement: I feel explaining the motivation for the use of the variable would be useful, especially because `server_definitions.json` is constructed from the python script. It would save future developers from having to read the python script.

Reply:
I added some more explanations where these values are actually used - is that enough?

Reply:
yes, the comments are helpful, thanks